tools/web3-plugin-example/src/custom_rpc_methods.ts

```diff
@@ -25,7 +26,24 @@ type CustomRpcApi = {
 
 export class CustomRpcMethodsPlugin extends Web3PluginBase<CustomRpcApi> {
 	public pluginNamespace = 'customRpcMethods';
+	public web3Middleware: Web3Middleware<CustomRpcApi> | undefined;
 
+	public constructor(testMiddleware = false){
+		super();
+
+		if(testMiddleware)
+			this.web3Middleware = new Web3Middleware<CustomRpcApi>();
+	}
+
+public link(parentContext: Web3Context): void {
```

Comment:
Could you please check the prettier config on your IDE? Looks like it is not correct code automatically